lua/nvim-treesitter/parsers.lua

```diff
@@ -74,7 +75,8 @@ list.rust = {
 list.ledger = {
   install_info = {
     url = "https://github.com/cbarrete/tree-sitter-ledger",
-    files = { "src/parser.c" }
+    files = { "src/parser.c" },
+    requires_generate_from_grammar  = true,
```

Comment:
ledger in on 0.19.x now

Reply:
that's not true. I just checked the C files

Reply:
https://github.com/cbarrete/tree-sitter-ledger/blob/609d5e5ab5955823b3faeaec8d2afc91860c639a/src/parser.c#L8